elementary/clients/slack/slack_message_builder.py

```diff
@@ -155,14 +155,12 @@ def get_slack_message(self) -> SlackMessageSchema:
         return SlackMessageSchema(**self.slack_message)
 
     @staticmethod
-    def prettify_and_dedup_list(
-        list_variation: Union[List[str], str]
-    ) -> Union[List[str], str]:
-        if isinstance(list_variation, str):
-            return prettify_json_str_set(list_variation)
-
-        elif isinstance(list_variation, list):
-            return ", ".join(set(list_variation))
-
-        else:
-            return list_variation
+    def prettify_and_dedup_list(str_list: Union[List[str], str]) -> str:
+        """
+        Receives a list of strings, either JSON dumped or not, dedups and sorts it, and returns it as a comma-separated
+        string.
+        This is useful for various lists we include in slack messages (owners, subscribers, etc)
+        """
```

Comment:
I removed `prettify_json_str_set` because it was confusing.
Made this function always return a string because that's the intention (it's for the owner list in slack messages)